future/transaction.go

```diff
@@ -534,7 +534,7 @@ func MakeApplicationCreateTx(
 	note []byte,
 	group types.Digest,
 	lease [32]byte,
-	rekeyTo types.Address, extraPages uint32) (tx types.Transaction, err error) {
+	rekeyTo types.Address) (tx types.Transaction, err error) {
 
```

Comment:
Nit: to reduce (a small amount of) code duplication, could you make the body of this function just call `MakeApplicationCreateTxWithExtraPages` with 0 extra pages?